starenchenkonv/first/task1.py

```diff
@@ -0,0 +1,8 @@
+new_list = [1,2,4,5.1,5.5,6,7.7,67.6,453.6,4356.56,5645.6]

```

Comment:
new_list = [1,2,4,5.1,5.5,6,7.7,67.6,453.6,4356.56,5645.6]
float_list = []


def get_floats_len(float_list, new_list):
    for index in new_list:
        if isinstance(index, float):
            float_list.append(index)
    print(len(float_list))
    return len(float_list)


get_floats_len(float_list, new_list)